symfony.lock

```diff
@@ -40,6 +40,9 @@
     "doctrine/common": {
         "version": "3.1.1"
     },
+    "doctrine/data-fixtures": {
+        "version": "1.5.0"
+    },
```

Comment:
Just a little remark. 

I know that you've been trying different approaches to solve fixture issues.
Remember that once you decide the approach or library to use you must remove the ones that finally are unused.
Not sure if this is the case and we finally are not using doctrine/data-fixture or if it is a requirement for alice bundle and this file is correct as it is, but please, as mentioned, make sure that we remove from our composer dependencies all the libraries we finally are not using.